presage/src/manager/registered.rs

```diff
@@ -719,6 +720,7 @@ impl<S: Store> Manager<S, Registered> {
             metadata: Metadata {
                 sender: self.state.data.service_ids.aci.into(),
                 sender_device: self.state.device_id(),
+                server_guid: None, // TODO: What value should this have?
```

Comment:
Signal Desktop has the following comment:

```
  // This should always be set for new messages, but older messages may not have them. We
  //   may not have these for outbound messages, either, as we have not needed them.
```

So, I guess it is ok to leave it empty.